### PR TITLE
untangle next deps

### PR DESCRIPTION
**Pull request by joelhooks**
share

### CODE REVIEW

File: apps/astro-party/src/pages/tutorials/[module]/[section]/[lesson]/index.tsx [hunk 1]
```diff
@@ -21,9 +21,9 @@ export const getStaticProps: GetStaticProps = async (context) => {
   const module = await getTutorial(params?.module as string)
   const section = await getSection(sectionSlug)
   const lesson = await getExercise(lessonSlug, false)
-  const lessonBody = lesson.body && (await serialize(lesson.body))
+  const lessonBody = lesson.body && (await serialize(String(lesson.body)))
```
Comment (joelhooks) [inline apps/astro-party/src/pages/tutorials/[module]/[section]/[lesson]/index.tsx:24]: upgrade to next-mdx-remote had stricter types

File: apps/testing-javascript/src/path-to-purchase/pricing.tsx [hunk 2]
```diff
@@ -11,14 +11,14 @@ import type {
   FormattedPrice,
 } from '@skillrecordings/commerce-server/dist/@types'
 import {CheckCircleIcon} from '@heroicons/react/outline'
-import {useDebounce} from '@skillrecordings/react'
```
Comment (joelhooks) [inline apps/testing-javascript/src/path-to-purchase/pricing.tsx:14]: the core of this is that dependencies on sass in various packages are causing nextjs in node_modules/pnpm to have two different versions

this makes the build break in mysterious all-day tears sorts of ways

File: packages/feedback-widget/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "build": "tsc --jsx react-jsx --outDir dist & pnpm build:css",
-    "build:css": "sass --style expanded src/styles.css dist/styles.css",
+    "build:css": "cp src/styles.css dist/styles.css",
```
Comment (joelhooks) [inline packages/feedback-widget/package.json:14]: I don't think we need sass generally

File: packages/player/package.json [hunk 4]
```diff
@@ -72,6 +72,9 @@
     "react-dom": "^17.0.2 < 18.0.0",
     "xstate": "^4.23.1 > 5"
   },
+  "transitivePeerDependencies": {
+    "sass": "^1.43.4"
+  },
```
Comment (joelhooks) [inline packages/player/package.json:77]: these can probably all be removed

File: packages/skill-api/package.json [hunk 5]
```diff
@@ -53,11 +53,11 @@
     "preact": "^10.7.2",
     "preact-render-to-string": "^5.2.0",
     "sanitize-html": "^2.7.0",
+    "styled-jsx": "5.1.1",
```
Comment (joelhooks) [inline packages/skill-api/package.json:56]: this resolved a similar issue where we had THREE pnpm/node_modules nextjs variants

File: packages/skill-api/package.json [hunk 6]
```diff
@@ -53,11 +53,11 @@
...
     "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@babel/cli": "^7.16.0",
-    "@babel/core": "^7.16.5",
```
Comment (joelhooks) [inline packages/skill-api/package.json:60]: or maybe it was this 😭

**Review by vojtaholik — APPROVED** [review]
🎉🎉🎉

**Pull request merged**